kalite/store/models.py

```diff
@@ -51,7 +51,7 @@ class StoreTransactionLog(DeferredCountSyncedModel):
     Model to track student 'expenditure' of points.
     """
 
-    # TODO-BLOCKER(rtibbles): Update this to "0.13.0" (or whatever the release version number is at the time this goes upstream)
+    # TODO(rtibbles): Update this to "0.13.0" (or whatever the release version number is at the time this goes upstream)
```

Comment:
Correct on the first. @jamalex should we just be deleting most of the store or nay?

Min-version is used on specific models for syncing purposes. We won't try to sync models that are 0.13 min-version to a 0.12 system, for example.